### PR TITLE
Libhandler system

### PR DESCRIPTION
**Pull request by dhil**
This PR adds libhandler as a system.

### CODE REVIEW

**Comment by jO-Osko** [comment]
Looks good to me. Since this already has changes from #26 I'll prefer to merge that one first.

**Comment by dhil** [comment]
Ah, that's a mistake. I probably branched on `pipes`...

**Comment by dhil** [comment]
My latest commit reroots the branch to `main`.

**Comment by dhil** [comment]
I've pushed the Docker images to dockerhub effecthandlers/effect-handlers.

File: Makefile [hunk 1]
```diff
@@ -50,6 +50,18 @@ ci_bench_links: sys_links
 	docker run -v $(shell pwd):/source $(DOCKERHUB):links \
 		make -C /source/benchmarks/links
 
+# libhandler
+sys_libhandler:
+	docker build -t $(DOCKERHUB):libhandler systems/libhandler
+
+bench_libhandler: sys_libhandler
+	docker run -it --init -v $(shell pwd):/source $(DOCKERHUB):libhandler \
+		make -C /source/benchmarks/libhandler
```
Comment (jO-Osko) [inline Makefile:59]: Can you add a (possibly empty)?
This will fix CI build.

Comment (dhil) [reply Makefile:59]: Done!

**Comment by jO-Osko** [comment]
Looks good, thanks!

**Pull request merged**